docs/custom-flows/multi-session-applications.mdx

```diff
@@ -1,74 +1,54 @@
+</Callout>
 
 A multi-session application is an application that allows multiple accounts to be signed in from the same browser at the same time. The user can switch from one account to another seamlessly. Each account is independent from the rest and has access to different resources.
 
-## Enable multi-session in your application
-
-To enable multi-session in your application, you need to configure it in the Clerk Dashboard.
-
-- Navigate to the [Clerk Dashboard](https://dashboard.clerk.com/last-active?path=sessions)
-- Select your application, then select **Sessions** in the navigation sidebar.
-- Toggle on **Multi-session handling**.
-- Select **Save changes**.
-
-## Add multi-session support to your application
-
-There are two main ways to add the multi-session feature to your application:
-- [Use Clerk components](#use-clerk-components) if you want to use Clerk's pre-built UI
-- [Build a *custom flow*](#build-a-custom-flow) if you want to rebuild the existing Clerk flow using the Clerk API.
+This guide provides you with the necessary information to build a custom multi-session flow using Clerk's API.
 
-### Use Clerk components
+To implement the multi-session feature to your application, you need to handle the following scenarios:
 
-The easiest way to add the multi-session feature to your application is by using Clerk's [`<UserButton />`](/docs/components/user/user-button) component.
+- [Switching between different accounts](#switch-between-sessions)
+- [Adding new accounts](#add-a-new-session)
+- [Signing out from one account, while remaining signed in to the rest](#sign-out-active-session)
+- [Signing out from all accounts](#sign-out-all-sessions)
 
-The following image demonstrates the `<UserButton />` component with two accounts signed in. The user can switch between the two accounts, add a new account, or sign out from one or all accounts.
-
-<Images
-  width={440}
-  height={479}
-  src="/docs/images/custom-flows/user-button-multisession.jpg"
-  alt="The User Button component with multiple accounts signed in."
-/>
-
-To learn how to use the `<UserButton />` component in your application, check out the [reference documentation](/docs/components/user/user-button).
-
-### Build a custom flow
-
-If the `UserButton />` component doesn't cover your needs, you can build a custom multi-session flow using Clerk's API.
+## Enable multi-session in your application
 
-To enable a multi-session instance, you need to handle the following scenarios:
+To enable multi-session in your application, you need to configure it in the Clerk Dashboard.
 
-- Switching between different accounts
-- Adding new accounts
-- Signing out from one account, while remaining signed in to the rest
+1. Navigate to the [Clerk Dashboard](https://dashboard.clerk.com/last-active?path=sessions)
```

Comment:
Same suggestions for consolidating these steps as above:
https://github.com/clerk/clerk-docs/pull/788/files#r1518068995